pkg/check/base.go

```diff
@@ -28,6 +29,10 @@ func (c *BaseCheck) Setup(ok string, nok string, metricName string, metricHelp s
 		Name: fmt.Sprintf("%s_duration", metricName),
 		Help: fmt.Sprintf("The duration of %s in ms", metricName),
 	}, labels)
+	c.SummaryMetric = promauto.NewSummaryVec(prometheus.SummaryOpts{
+		Name: fmt.Sprintf("%s_summary", metricName),
+		Help: fmt.Sprintf("The duration of resolver lookups %s in ms and percentiles", metricName),
```

Comment:
Shouldn't there be the an Objectives map to have quantiles generated?
https://github.com/prometheus/client_golang/blob/master/prometheus/summary.go#L116-L121

Reply:
I thought the Summary still had a defaultObjectives map of .5, .9 and .99 but it looks like it was deprecated. Will fix it soon.